doc/releases/changelog-dev.md

```diff
@@ -128,6 +125,10 @@
 
   ```
 
+* When new operator arithmetic is enabled, `qml.Hamiltonian` is now an alias for `qml.ops.LinearCombination`.
```

Comment:
Could this potentially be a Breaking Change if someone creates a `qml.Hamiltonian` and expects it to be an instance of the old `Hamiltonian`?

Reply:
Good point. I'll move it to the breaking changes section.